migrations/20160606164500-add-expenses-table.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+const status = require('../server/constants/expense_status');
+
+module.exports = {
```

Comment:
Right. Well no code is actually using that model but it's true that it's still created.